src/Storage.java

```diff
@@ -0,0 +1,16 @@
+import com.sun.istack.internal.Nullable;
+
+import java.util.HashMap;
+
+public class Storage<T, K> extends HashMap{
+
+	@Override
```

Comment:
create your own implementation

Reply:
I deleted extending and change logic and change hashCode in Pair.